### ISSUE TITLE
bash syntax vs default shell

### ISSUE THREAD

**Issue by benclifford** [Parsl/parsl#1301]
Using htex, localprovider, sshchannel, approximate commands like this are attempted to launch the process worker pool

```
ssh 172.17.0.5 "bash /home/benc/parsl/src/parsl/runinfo/032/submit_scripts/parsl.auto.1569309957.2783496.sh &> /home/benc/parsl/src/parsl/runinfo/032/submit_scripts/parsl.auto.1569309957.2783496.sh.out &"
```

These commands are initially interpreted by the user's default shell, not by `bash`.

In the case that the default shell does not accept that redirection syntax (for example, bash running as plain /bin/sh), strange errors occur.

A workaround is to ensure the remote user's default shell is `bash`.

**Comment by benclifford** [comment]
Actually I think that workaround won't work - the documentation for subprocess.Popen says

> On POSIX with shell=True, the shell defaults to /bin/sh. 

in https://docs.python.org/3/library/subprocess.html#subprocess.Popen

**Comment by benclifford** [comment]
This should be fixed by PR #1452
